applications/EigenSolversApplication/custom_python/add_custom_solvers_to_python.cpp

```diff
@@ -40,33 +40,46 @@ void AddCustomSolversToPython()
 	typedef LinearSolver<SparseSpaceType, LocalSpaceType> LinearSolverType;
 	typedef DirectSolver<SparseSpaceType, LocalSpaceType> DirectSolverType;
 
+	// --- direct solvers
+
 	using SparseLUSolver = EigenDirectSolver<SparseLU, SparseSpaceType, LocalSpaceType>;
 	class_<SparseLUSolver, bases<DirectSolverType>, boost::noncopyable>
 		("SparseLUSolver", init<>())
-		.def(init<Parameters>());
+		.def(init<Parameters>())
+	;
 
-	#if defined EIGEN_USE_MKL_ALL
+	#if defined MKL
```

Comment:
this is not very verbose, I would recommend using sth like `USE_MKL_EIGEN`

Reply:
Does it make sense, that each app defines an own MKL-available variable?

Reply:
Since there is no global solution to include MKL I would say yes ... my opinion though
at least I would rename it to `USE_MKL` if you want a global variable

Reply:
AFAIK this is the only application that uses MKL at the moment. As already mentioned it would be the best to have a global solution.